package.json

```diff
@@ -36,9 +37,10 @@
     "monocle-ts": "^2.3.3",
     "mustache": "^4.0.1",
     "newtype-ts": "^0.3.4",
+    "popper": "^1.0.1",
```

Comment:
What pulls this in?  also is this intended to be https://www.npmjs.com/package/popper.js or https://www.npmjs.com/package/popper I'm confused